crates/services/gas_price_service/src/v0/service.rs

```diff
@@ -203,7 +203,6 @@ mod tests {
         l2_block: mpsc::Receiver<BlockInfo>,
     }
 
-    #[async_trait::async_trait]
     impl L2BlockSource for FakeL2BlockSource {
         async fn get_l2_block(&mut self) -> GasPriceResult<BlockInfo> {
```

Comment:
Does the `async` still work here? I'm not sure if it desugars to the same.

Reply:
yes

Reply:
Yeah. I'm playing around with it in playground. Works for me.